mmv1/third_party/terraform/resources/resource_apigee_env_keystore_alias_pkcs12.go

```diff
@@ -0,0 +1,627 @@
+				Optional:    true,
+				Computed:    true,
+				Description: `Chain of certificates under this alias.`,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cert_info": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							Description: `List of all properties in the object.`,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"basic_constraints": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 basic constraints extension.`,
+									},
+									"expiry_date": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 notAfter validity period in milliseconds since epoch.`,
+									},
+									"is_valid": {
+										Type:     schema.TypeString,
+										Optional: true,
+										Computed: true,
+										Description: `Flag that specifies whether the certificate is valid. 
+Flag is set to Yes if the certificate is valid, No if expired, or Not yet if not yet valid.`,
+									},
+									"issuer": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 issuer.`,
+									},
+									"public_key": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `Public key component of the X.509 subject public key info.`,
+									},
+									"serial_number": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 serial number.`,
+									},
+									"sig_alg_name": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 signatureAlgorithm.`,
+									},
+									"subject": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 subject.`,
+									},
+									"subject_alternative_names": {
+										Type:        schema.TypeList,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 subject alternative names (SANs) extension.`,
+										Elem: &schema.Schema{
+											Type: schema.TypeString,
+										},
+									},
+									"valid_from": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 notBefore validity period in milliseconds since epoch.`,
+									},
+									"version": {
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 version.`,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"password": {
```

Comment:
Is this also output-only? If so, remove `optional: true`, otherwise, since we don't have a update function for this resource (I assume this resource is immutable), we need to add Forcenew. Or consider adding update functionality for this resource.